Marlin/src/lcd/menu/menu_main.cpp

```diff
@@ -458,6 +459,17 @@ void menu_main() {
     });
   #endif
 
+  #if DISABLED(DISABLE_ENCODER) && ENABLED(ADVANCED_PAUSE_FEATURE)
+    #if E_STEPPERS == 1 && DISABLED(FILAMENT_LOAD_UNLOAD_GCODES)
+      YESNO_ITEM(MSG_FILAMENTCHANGE,
+        menu_change_filament, nullptr,
+        GET_TEXT_F(MSG_FILAMENTCHANGE), (const char *)nullptr, F("?")
+      );
+    #else
+      SUBMENU(MSG_FILAMENTCHANGE, menu_change_filament);
+    #endif
+  #endif
```

Comment:
I'm not fond of having repeated code that must be altered exactly in tandem with some identical block in another place, especially when no note is added to the code to alert developers of this situation. I will see about making this editable in a single location.

Reply:
I''m cringing at this too. Of only we could use macro inside macro substitutes... Maybe turn it into `void insert_fimchange_item()`?